modules/end-user-guide/pages/selecting-an-in-browser-ide-for-one-new-workspace.adoc

```diff
@@ -0,0 +1,18 @@
+:_content-type: PROCEDURE
+:description: Selecting an in-browser IDE for one new workspace
+:keywords: select-IDE
+:navtitle: Selecting an in-browser IDE for one new workspace
+:page-aliases: selecting-an-ide-by-using-a-url-parameter.adoc
+
+[id="selecting-an-in-browser-ide-for-one-new-workspace_{context}"]
+= Selecting an in-browser IDE for one new workspace
+
+You can select your preferred in-browser IDE when using a URL for starting a new workspace. This way, each developer using {prod-short} can start a workspace with a clone of the same project repository and yet with the personal choice of the in-browser IDE.
```

Comment:
```suggestion
You can select your preferred in-browser IDE when using a URL for starting a new workspace. This way, each developer using {prod-short} can start a workspace with a clone of the same project repository but use their preferred in-browser IDE.
```

Reply:
:) I'm trying to avoid using `their` here.